app/services/friendly_password_generator.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class FriendlyPasswordGenerator
```

Comment:
I'm not sure we need it. Maybe we can remove it. 

We just need a password to create a new student account. 

Maybe it will come from the CSV or be generated by the system and send out an email... There are too many questions at the moment. 

What do you think?

Reply:
It can be related to that, too:

https://github.com/rubyforgood/stocks-in-the-future/issues/311